src/hal/utils/halcmd_main.c

```diff
@@ -279,10 +281,18 @@ int main(int argc, char **argv)
     {
 	char cmdline[200];
```

Comment:
There is a MAX_CMD_LEN already defined at 1024, which should solve the problem both ways